theme/static/js/custom.js

```diff
@@ -422,6 +422,14 @@ $(document).ready(function () {
         this.box.insertAfter(this.input).css({top: 35, left: 0});
     };
 
+    // Prevent clicking on list items dismissing the modal window
+    $(".autocomplete-light-widget > input.autocomplete").each(function (i, el) {
+        $(el).yourlabsAutocomplete()
```

Comment:
such a weird method name, but that's what the module provides

Reply:
Indeed. Here are the docs for this section for future reference: https://django-autocomplete-light.readthedocs.io/en/2.2.9/script.html